WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PersonsController < ApplicationController
+  def index
+    params[:region] ||= "all"
+
+    respond_to do |format|
+      format.html
+      format.js do
+        persons = Person.joins("JOIN Countries ON countryId = Countries.id")
+        if params[:region] != "all"
+          persons = persons.where("countryId = :region OR continentId = :region", region: params[:region])
+        end
+        if params[:search].present?
+          persons = persons.where("rails_persons.name LIKE :input OR wca_id LIKE :input", input: "%#{params[:search]}%")
+        end
+        persons = persons.order(:name, :countryId)
+
+        render json: {
+          total: persons.count,
+          rows: persons.limit(params[:limit]).offset(params[:offset]).map do |person|
+            {
+              name: view_context.link_to(person.name, "/results/p.php?i=#{person.wca_id}"),
+              wca_id: person.wca_id,
+              country: person.country.name,
+              competitions_count: person.competitions.count,
+              podiums_count: person.results.where(roundId: [:f, :c], pos: [1, 2, 3]).where('best > 0').count,
```

Comment:
Can we share this code with https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/models/competition.rb#L528-L531? How about a `podium` scope on the Result class?